README.md

```diff
@@ -74,8 +74,8 @@ Once you have confirmed access, you can run the following command to download th
 ```bash
 # Get your HF token from https://huggingface.co/settings/tokens
 
-# llama3 tokenizer.model
-python torchtitan/datasets/download_tokenizer.py --repo_id meta-llama/Meta-Llama-3-8B --tokenizer_path "original" --hf_token=...
+# llama3 or 3.1 tokenizer.model
+python torchtitan/datasets/download_tokenizer.py --repo_id meta-llama/Meta-Llama-3-8B --local_dir "torchtitan/datasets/tokenizer/" --tokenizer_path "original" --hf_token=...
```

Comment:
may I ask why adding this? this seems to be the default location

Reply:
I think we do need to let users they can specify where to store the tokenizer? Or we can say it somewhere?